### PR TITLE
Fix usage in README

### PR DESCRIPTION
**Pull request by szepeviktor**
Command was missing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ composer require tomasvotruba/class-leak --dev
 Pass directories you want to check:
 
 ```bash
-vendor/bin/class-leak bin src
+vendor/bin/class-leak check bin/ src/
```
Comment (szepeviktor) [inline README.md:18]: `bin/` Show that bin is a directory. Shell autocomplete (TAB key) does this also.

File: README.md [hunk 2]
```diff
@@ -25,5 +25,5 @@ Make sure to exclude `/tests` directories, to keep reporting classes that are us
 Many types are excluded by default, as they're collected by framework magic, e.g. console command classes. To exlude another class, e.g. your interface collector, use `--skip-type`:
 
 ```bash
-vendor/bin/class-leak check bin src --skip-type="App\\Contract\\SomeInterface"
+vendor/bin/class-leak check bin/ src/ --skip-type='App\Contract\SomeInterface'
```
Comment (szepeviktor) [inline README.md:28]: In shell backslashes do not work in single-quotes.

File: README.md (hunk 1 above)
Comment (TomasVotruba) [reply README.md:18]: The name is needed, you're right :+1: 

But I don't like the extra slashes as extra writing.

File: README.md (hunk 2 above)
Comment (TomasVotruba) [reply README.md:28]: This depends on OS and particular shell interpretter. The `--skip-type='App\Contract\SomeInterface'` is also good :+1:

File: README.md (hunk 1 above)
Comment (szepeviktor) [reply README.md:18]: This is too personal for me.
I am so sorry.

**Pull request closed without merging**